test/TestProjects/TypeSchemaMapping/Customized/CustomFruitEnum.cs

```diff
@@ -1,9 +1,12 @@
 using Azure.Core;
 
-namespace AnotherCustomNamespace
+namespace NamespaceForEnums
 {
     [CodeGenSchema("Fruit")]
-    internal partial struct CustomFruitEnum
+    internal enum CustomFruitEnum
```

Comment:
Because `enum` types can't be partial you have to copy all the values to remap the enum.

Another cool feature is that generator would switch between generating extensible or normal enum based on what type `[CodeGenSchema]` is applied to.